src/backend/routers/para.test.ts

```diff
@@ -44,3 +77,52 @@ test("createPara", async (t) => {
       .some((mail) => mail.subject?.includes("confirmation"))
   );
 });
+
+test("duplicateEmail", async (t) => {
+  const { db } = await getTestServer(t, { authenticateAs: "para" });
+
+  await t.throwsAsync(() => {
+    return db
+      .insertInto("user")
+      .values({
+        first_name: "Foos",
+        last_name: "Bar",
+        email: "john@example.com",
+        role: "staff",
+      })
+      .execute();
+  });
+});
```

Comment:
it's not entirely clear what this is testing at a quick glance

if I was writing this test, I would:
- replace `john@example.com` with `seed.para.email` or whatever's returned from the fixture seed (I think this is why it's throwing?)
- edit the test name to make the goal more clear
- move this test to something like `src/backend/db/migrations/1_initial-migrations.test.ts`, since, as written, this isn't touching any trpc procedures. Alternatively switch out the database call for a procedure call

Reply:
This test is to test for non-duplicate email for `user` in the db. We can move it to that route if it's more appropriate.

Reply:
I think if we address the first two points above and swap the database call for a procedure call we should be good

(the convention in this repo should be to write as high-level tests as possible that replicate how the frontend interacts with the data persistence layer)